static/app/views/replays/replayTable/tableCell.tsx

```diff
@@ -516,18 +517,28 @@ export function OSCell({replay, showDropdownFilters}: Props) {
 }
 
 export function BrowserCell({replay, showDropdownFilters}: Props) {
-  const {name, version} = replay.browser ?? {};
+  const {name, version} = replay.browser;
   const theme = useTheme();
   const hasRoomForColumns = useMedia(`(min-width: ${theme.breakpoints.large})`);
 
   if (replay.is_archived) {
     return <Item isArchived />;
   }
+
+  if (name === null && version === null) {
```

Comment:
first i tried looking at the os or sdk info.. then the `platform` field.. both of those didn't work because we don't have full mappings of 'platforms that have a browser'.

So, normally we'd want to look for mobile platforms, but `java` (via react-native) is one combo that's reported back, but not in the list as a mobile-specific platform. 

So best we can do in this case where a mix of projects is selected is to just show icons for what we have, and if there's no icon that's ok.